### PR TITLE
Add setup instructions to README.md

### LINKED ISSUE #5: [8pt] Setup instructions
- [ ] [8pt] Document steps to get the project up and running and add to project README.md

### PR DESCRIPTION
**Pull request by maxwentmissing**
I copied the instructions Greg wrote in the Discord groupchat into the README.md file. Can you guys please test it out and take a look?

Closes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ## Setup Instructions
 
+The below instrrrctions are the steps needed to download all of the prerequesites on Ubuntu 20.04. It can be either a real Ubuntu or Ubuntu on WSL. Follow these instructions to set up Ubuntu 20.04 on WSL. https://wiki.ubuntu.com/WSL
```
Comment (z1868762) [inline README.md:9]: Typo. pls fix ;-;

**Review by z1868762 — CHANGES_REQUESTED** [review]
Looks Good Except for this typo. Pls fix

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 ## Setup Instructions
 
-The below instrrrctions are the steps needed to download all of the prerequesites on Ubuntu 20.04. It can be either a real Ubuntu or Ubuntu on WSL. Follow these instructions to set up Ubuntu 20.04 on WSL. https://wiki.ubuntu.com/WSL
+The below instructions are the steps needed to download all of the prerequesites on Ubuntu 20.04. It can be either a real Ubuntu or Ubuntu on WSL. Follow these instructions to set up Ubuntu 20.04 on WSL. https://wiki.ubuntu.com/WSL
```
Comment (zgjs) [inline README.md:9]: Hey @z1868762 I fixed the typo! :)

**Review by z1868762 — APPROVED** [review]
image

**Pull request merged**